codecs/imagequant/Makefile

```diff
@@ -38,5 +55,5 @@ $(CODEC_DIR):
 	curl -sL $(CODEC_URL) | tar xz --strip 1 -C $@
 
 clean:
-	$(RM) $(OUT_JS) $(OUT_WASM)
+	$(RM) $(OUT_JS) *.wasm
```

Comment:
Same as above, once ENVIRONMENT is made a variable dependency, I think you can just put them into the same var like `OUT_JS := imagequant.js imagequant_node.js` and then this part can be reverted.